reana_workflow_controller/consumer.py

```diff
@@ -126,14 +126,14 @@ def on_message(self, body, message):
                     f"Event for workflow {workflow_uuid} that doesn't exist in DB received:\n"
                     f"{body}\nIgnoring..."
                 )
-        except REANAWorkflowControllerError as rwce:
-            logging.error(rwce, exc_info=True)
```

Comment:
As far as I can tell, this exception cannot occur because it is raised by `_get_workflow_engine_pod_logs` and `_delete_workflow_job`, but both of them have surrounding `try ... catch`es when called that catch this exception type.